app/forms/AssistanceDetailsForm.scala

```diff
@@ -60,7 +43,66 @@ object AssistanceDetailsForm {
                    needsSupportForOnlineAssessment: String,
                    needsSupportForOnlineAssessmentDescription: Option[String],
                    needsSupportAtVenue: String,
-                   needsSupportAtVenueDescription: Option[String]
-  )
+                   needsSupportAtVenueDescription: Option[String]) {
+
+    def exchange: AssistanceDetails = {
+      AssistanceDetails(
+        hasDisability,
+        hasDisabilityDescription,
+        guaranteedInterview.map {
```

Comment:
I will add this to the technical debt document. I would like to improve all that code at some point.
https://confluence.tools.tax.service.gov.uk/display/FSET/Technical+Debt